charts/jenkins/README.md

```diff
@@ -176,19 +177,12 @@ Each type supports a different set of configurable attributes, defined by [the c
 
 To make use of the NetworkPolicy resources created by default, install [a networking plugin that implements the Kubernetes NetworkPolicy spec](https://kubernetes.io/docs/tasks/administer-cluster/declare-network-policy#before-you-begin).
 
-For Kubernetes `v1.5` & `v1.6`, you must also turn on NetworkPolicy by setting the DefaultDeny namespace annotation.
-Note: this will enforce policy for _all_ pods in the namespace:
-
-```console
-kubectl annotate namespace default "net.beta.kubernetes.io/network-policy={\"ingress\":{\"isolation\":\"DefaultDeny\"}}"
-```
```

Comment:
should be safe to remove docs for such old versions of kubernetes
https://en.wikipedia.org/wiki/Kubernetes#Release_versions

Reply:
agree